x/icqoracle/keeper/genesis.go

```diff
@@ -10,6 +13,10 @@ import (
 func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
 	for _, tokenPrice := range genState.TokenPrices {
+		tokenPrice.SpotPrice = math.LegacyZeroDec()
+		tokenPrice.UpdatedAt = time.Time{}
+		tokenPrice.QueryInProgress = false
```

Comment:
This function is only called on genesis, where we don't even have IBC set up. In my mind these values are definitely stale at this point.

I guess there's no real reason to reset `SpotPrice` & `UpdatedAt` because there's also `PriceExpirationTimeoutSec` to make sure they are not used when stale. As for `QueryInProgress`, there's definitely no IBC at this point so I feel confident we can reset it and prevent potential issues down the road.